lib/parsers/txt.mjs

```diff
@@ -175,7 +177,8 @@ export async function parse (rawText, filename) {
     },
     references: {
       rfc2119: false,
-      rfc8174: false
+      rfc8174: false,
+      type: null
```

Comment:
This object is used to mark whether or not there is a reference to something. Not for marking general information about the file.